.github/renovate.json5

```diff
@@ -0,0 +1,64 @@
+{
+  "extends": [
+    "config:recommended",
+    ":approveMajorUpdates",
+  ],
+  // Create PRs on Monday midnight (UTC)
+  "schedule": [
+    "after 12am on Monday"
+  ],
+  "minimumReleaseAge": "7 days",
+  "vulnerabilityAlerts": {
+    "labels": [
+      "type:security"
+    ],
+    "minimumReleaseAge": "1 days"
+  },
+  "labels": ["dependencies"],
+  "dependencyDashboardLabels": ["type: process"],
+  "dockerfile": {
+    "enabled": false
+  },
+  "semanticCommits": true,
+  "pip-compile": {
+    "enabled": true,
+    "fileMatch": ["(^|/)requirements\\.in$"],
+  },
+  "constraints": {
+    "python": "~=3.11.0"
+  },
+  "ignorePaths": [
+    "demos/**",
+    "security/language-vulns/**"
+  ],
+  "packageRules": [
```

Comment:
Please re-add the per-language grouping rules. They should look like this:

```
    {
      "matchCategories": ["golang"],
      "matchUpdateTypes": ["minor", "patch", "digest", "lockFileMaintenance"],
      "groupName": "golang",
      "automerge": true
    },
```
one rule for each matchCategories: python,golang, java,js,dotnet,docker

Note that lockFileMaintenance is new, and i'm making corresponding PRs for the other repos now.

Reply:
What is a reason for duplicated rules? There is already https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/pull/977/files#diff-20ab1190d08a53a3012bc191ed56205c8f0ffb8fa1715e9d36ecfd1d2ea8a790R40-R43